TUTORIAL-ASSETS.md

```diff
@@ -247,40 +245,33 @@ Hit continue and after a bit more processing you should now see your Assets insi
 
 ## Editing your Assets
 
-Now that you've imported your Assets, you can make any changes you wish to their position or orientation. To do this, you can either click on the model directly, or select from the Asset List panel by selecting the Asset Category.
-
-![Image showing the Asset Category List](images/cadtutorial/asset_list_category.png)
-
-Clicking on this will show you your list of Assets of the specified category. Here you can select them, which will move the camera and highlight the selected Asset, and also delete individual ones as required.
+Now that you've imported your Assets, you can make any changes you wish to their position or orientation. To do this, you can either click on the model directly, or select it from the Asset List panel. In the Asset List panel you can select your Assets, which will highlight the selected one, and also delete individual ones as required.
 
 ![Image showing the Asset Category List](images/cadtutorial/asset_list_highlight.png)
 
-You can also click and drag selected Assets by clicking the ![2D Edit](images/cadtutorial/2d_edit.png) button and clicking and dragging the gizmo that appears.
+You can also click and drag selected Assets by clicking the dragging the gizmo.
 
 ![Image showing desk movement](images/cadtutorial/desk_move.gif)
 
 Selecting an Asset also shows the Asset Properties panel, which allows you to edit the selected Asset in more detail.
```

Comment:
Suggest moving the Asset Properties section before the click and drag section, as it reads better that way round.